spec/tree_spec.rb

```diff
@@ -5,10 +5,80 @@
   it 'should be a Class' do
     expect(described_class.is_a? Class).to eq true
   end
+
+  let(:fuji) {Tree.new}
+
+  it 'has a height, age, apples, alive' do
+    expect(fuji.height).to eq(1)
+    expect(fuji.age).to eq(0)
+    expect(fuji.apples).to be_empty
+    expect(fuji.alive).to be(true)
+  end
+
+  it 'does not have apples when initialized' do
+    expect(fuji.any_apples?).to eq(false)
+  end
+
+  it 'is not dead when initialized' do
+    expect(fuji.dead?).to eq(false)
+  end
+
+  it 'ages' do
+    fuji.age!    
+    expect(fuji.height).to eq(2)
+    expect(fuji.age).to eq(1)
+    expect(fuji.apples).to be_empty
+    expect(fuji.alive).to be(true)
+  end
+
+  it 'stops growing after it is 20 feet tall' do
```

Comment:
looks good but keep in mind using context to clean `it defs` like this and make them more readable.